Lib/mimetypes.py

```diff
@@ -722,18 +722,23 @@ def _main(args=None):
 
     args, help_text = _parse_args(args)
 
+    results = []
     if args.extension:
         for gtype in args.type:
             guess = guess_extension(gtype, not args.lenient)
             if guess:
-                return str(guess)
-            sys.exit(f"error: unknown type {gtype}")
+                results.append(str(guess))
+            else:
+                sys.exit(f"error: unknown type {gtype}")
+        return '\n'.join(results)
     else:
         for gtype in args.type:
             guess, encoding = guess_type(gtype, not args.lenient)
             if guess:
-                return f"type: {guess} encoding: {encoding}"
-            sys.exit(f"error: media type unknown for {gtype}")
+                results.append(f"type: {guess} encoding: {encoding}")
+            else:
+                sys.exit(f"error: media type unknown for {gtype}")
+        return '\n'.join(results)
     return help_text
```

Comment:
Is `return help_text` line used? 

Either:

- delete `return help_text` (that branch of the code is never used)
- change `return help_text` to `return "\n".join(results)` and delete the other `return` calls at the end of each `for type in args.type` block.